### ISSUE TITLE
Enh: Garbage

### ISSUE THREAD

**Issue by acs-ferreira** [humhub/humhub#2554]
One thing that could be very usefulis a garbage.

When a user deletes something (a post, a file, a poll...) the possibility to keep it in a configurable delay (1 day, 30days...) and that he have the possibility to restore it.

Something like on file clouds like Google Drive or OneDrive.

This could be useful on **private networks**.

Simple idea for future versions.

**Comment by Felli** [comment]
So in basic terms, the ability to be able to archive content like how Spaces are able to? If I'm not mistaken such things have been discussed before a number of times.

**Comment by acs-ferreira** [comment]
Yes, archive them but for a limited time. I didn't found anything related...
